Ternary Search Tree/Tests/TernarySearchTreeTests.swift

```diff
@@ -10,14 +10,23 @@ import XCTest
 
 class TernarySearchTreeTests: XCTestCase {
     
-    let treeOfStrings = TernarySearchTree<String>()
+
     let testCount = 30
     
     func testCanFindStringInTree() {
         var testStrings: [(key: String, data: String)] = []
+        let treeOfStrings = TernarySearchTree<String>()
+
         for _ in (1...testCount) {
```

Comment:
You could extract **testStrings** and **treeOfStrings** into properties of XCTestCase, and perform a setup to generate the tree. That way each test focuses on what it's testing.